### PR TITLE
make this package installable by pip

### LINKED ISSUE #98: packaging - creating an installable version of the software

### PR DESCRIPTION
**Pull request by prjemian**
- close #98

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  { name="Pete Jemian", email="prjemian@gmail.com" },
+]
+maintainers = [
+  { name="Fanny Rodolakis", email="rodolakis@anl.gov" },
+  { name="Pete Jemian", email="prjemian@gmail.com" },
+]
+dynamic = ["version"]
+readme = "README.md"
+requires-python = ">=3.8"
+keywords = ["bluesky", "databroker", "tiled"]
```
Comment (rodolakis) [inline pyproject.toml:27]: catalog?

**Review by rodolakis — CHANGES_REQUESTED** [review]
I would add catalog in the keyword. It looks good to me however I'd like you to walk me through the process sometime next week. There is going to be a lot of conflict with the PRs I put in yesterday though, how are you going to resolve that?

File: pyproject.toml (hunk 1 above)
Comment (prjemian) [reply pyproject.toml:27]: Sure.  Any other keywords?

Comment (rodolakis) [reply pyproject.toml:27]: Not that I can think of.

**Comment by prjemian** [comment]
Knew this would be some trouble.  I'm working through those now.  Time to improve my skills, it seems.  Might be better to push the other PRs through before I try to resolve the merge conflicts.

**Comment by rodolakis** [comment]
There is only one left, and I submitted it back for review. It was just formatting so it should not take long/.

**Comment by rodolakis** [comment]
All good, goo

> Knew this would be some trouble. I'm working through those now. Time to improve my skills, it seems. Might be better to push the other PRs through before I try to resolve the merge conflicts.

All good, good luck with that!

**Comment by prjemian** [comment]
Easier to start over than manage the various conflicts.  I'll add catalog to the keywords.

**Comment by prjemian** [comment]
See #152

**Pull request closed without merging**